src/datatypes/map.c

```diff
@@ -298,6 +298,25 @@ int Map_Compare
 	return 0;
 }
 
+// assumption: a and b are maps
```

Comment:
Improve comment,
the comment should contain an explanation on how the merge is performed in addition to an example.